File: src/main/java/com/taranovski/leetcode/solutions/easy/leetcode0001/java/Solution.java
package com.taranovski.leetcode.solutions.easy.leetcode0001.java;

/**
 * Created by Alyx on 27.09.2018.
 */
class Solution {
    public int[] twoSum(int[] nums, int target) {
        final int length = nums.length;
        for (int i = 0; i < length; i++) {
            for (int j = 0; j < length; j++) {
                if (i != j) {
                    if (nums[i] + nums[j] == target) {
                        return new int[]{i, j};
                    }
                }
            }
        }
        return null;
    }
}

File: src/main/java/com/taranovski/leetcode/solutions/easy/leetcode0001/kotlin/Solution.kt
package com.taranovski.leetcode.solutions.easy.leetcode0001.kotlin

/**
 * Created by Alyx on 27.09.2018.
 */
class Solution {
    fun twoSum(nums: IntArray, target: Int): IntArray? {
        val length = nums.size
        for (i in 0..length - 1) {
            for (j in 0..length - 1) {
                if (i != j) {
                    if (nums[i] + nums[j] == target) {
                        return intArrayOf(i, j)
                    }
                }
            }
        }
        return null
    }
}
